### DISCUSSION TITLE
swiping menu also moves photos

### DISCUSSION THREAD

**Discussion by andrewbanchich** [ente-io/ente#12322] [Enhancements]
### Description

On Android, swiping the menu from the left works, but it also moves the photos view to the right during this action. It would be nice if the photos didn't move when opening the menu.

### Version

0.9.30

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - Android

**Comment by AmanRajSinghMourya** [comment]
This is currently intentional: swiping right on the first Photos tab overscrolls the gallery and opens the settings drawer. The movement is part of that custom gesture rather than a regression or Android’s Back gesture. Since the request is to change the interaction so the drawer opens over stationary content, we’ll track it as a UX enhancement/feature request instead of a bug.
